CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 - New `/node/index/strategies` endpoint, which will list all indexing strategies currently installed alongside a description of how far each has progressed.
 - When starting a host subprocess, applications may now pass data to its standard input. Additionally, the process' output is captured and logged by CCF (#5056).
+- `CCF_IGNORE_FIRST_SIGTERM` environment variable. When set, will cause a node to ignore the first `SIGTERM` it receives, but the `/node/process` endpoint will return `{"stop_notice": true}`. A second `SIGTERM` will cause the process to shut down as normal. This can be useful in orchestration settings where nodes receive unsollicited signals that the operator wishes to react to.
```

Comment:
Should this be a configuration file parameter instead of environment variable? Don't think it's a good idea to pass settings in different ways

Reply:
I'll make it a config parameter.